_pages/CodeOfConduct.html

```diff
@@ -96,7 +96,10 @@ <h2>Reporting</h2>
             <p>You can make a report either personally or anonymously.
               All reports will be handled with discretion.</p>
 
-            <p>You can make a personal report directly to *****.</p>
+            <p>You can make a personal report directly to one of the two
+              contact people on site at each event. Their details will be
+              shared at the start of the event. You can also contact <a href="https://twitter.com/gedankenstuecke">Bastian Greshake Tzovaras (bgreshake@googlemail.com)</a>
```

Comment:
I would end the <a> tag at his name, before the e-mail address. Otherwise, it looks like clicking the link will take you to your mail client.